patching/plugin-commands-patching/src/patchRemove.ts

```diff
@@ -64,6 +64,7 @@ export async function handler (opts: PatchRemoveCommandOptions, params: string[]
       delete rootProjectManifest.pnpm!.patchedDependencies![patch]
       if (!Object.keys(rootProjectManifest.pnpm!.patchedDependencies!).length) {
         delete rootProjectManifest.pnpm!.patchedDependencies
+        await fs.rmdir(patchFile.replace(`${patch}.patch`, ''))
```

Comment:
it should be removed only if the directory is empty.

And to get the parent dir you can use `path.dirname(patchFile)`

Reply:
According to my understanding, when patchedDependencies has no value, the corresponding patch folder should be empty at this time, right?

Reply:
No, I think that is unsafe to do. We can't remove patch files created by the user. Imagine the user just wants to test the project without the patches. They remove the field temporarily. They don't want to loose the patch files. This is especially bad if the patch file was not committed previously. They might loose their changes.

Reply:
You are right, I will judge whether the patch folder is empty to decide whether the folder should be removed.